tsl/test/expected/continuous_aggs_bgw.out

```diff
@@ -358,6 +364,12 @@ CREATE VIEW test_continuous_agg_view
     AS SELECT time_bucket('2', time), SUM(data) as value
         FROM test_continuous_agg_table
         GROUP BY 1;
+SELECT add_refresh_continuous_aggregate_policy('test_continuous_agg_view', NULL, -2::integer, '12 h'::interval);
```

Comment:
I thought we weren't going to allow negative intervals.What does it mean to have a negative end interval (is it going into the future?)

Reply:
resolving as we decided to keep this for now.